BusBoard.ConsoleApp/Program.cs

```diff
@@ -1,15 +1,33 @@
 ﻿using System;
-using System.Collections.Generic;
 using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
+using System.Web.UI.HtmlControls;
 
 namespace BusBoard.ConsoleApp
-{
-  class Program
-  {
-    static void Main(string[] args)
+{ 
+    class Program
     {
+        static void Main()
+        {
+            while (true)
+            {
+                Console.Write("Stop code: ");
+                var stopPoint = Console.ReadLine();
+                if (string.IsNullOrEmpty(stopPoint))
+                {
+                    break;
+                }
+
+                var busEntries = ApiHelper
+                    .ApiGet<BusEntry>("https://api.tfl.gov.uk/", $"StopPoint/{stopPoint}/Arrivals").ToList();
+
+                busEntries = busEntries.OrderBy(busEntry => busEntry.expectedArrival).ToList();
+                busEntries = busEntries.GetRange(0, 5);
+
+                foreach (var entry in busEntries)
+                {
+                    Console.WriteLine(entry);
```

Comment:
Could use `String.Join` here with just one `WriteLine` instead of the `foreach`. That will reduce nesting 🙂